metadata_mapper/mappers/mapper.py

```diff
@@ -639,13 +639,14 @@ def enrich_subject(self):
         """
         # normalize subject
         subjects = self.mapped_data.get('subject', [])
+
         if isinstance(subjects, str):
             subjects = [subjects]
         if not subjects:
             subjects = []
         subjects = [
-            subject.get('name') for subject in subjects
-            if isinstance(subject, dict)
+            subject.get('name') if isinstance(subject, dict) else subject
+            for subject in subjects
```

Comment:
Rather than changing this in the enrichment, could you update the mapper to make subject conform to this `[{'name': <subject>}]` data structure? 

I know it seems strange to build this data structure only to have an enrichment remove the data structure, but this algorithm change here in the enrichments would require some data analysis across every collection running the `enrich_subject` enrichment.

Reply:
Sure thing.